CHANGELOG.md

```diff
@@ -4,44 +4,56 @@
 
 ### New features
 
-#### Create custom width container classes
+#### Set a specific width for wrappers
 
-You can now create custom page width container classes using the `govuk-width-container` mixin. You do this by passing in the required maximum width of the container.
+You can now:
+
+- create your own Sass classes with specific widths
+- add those classes to the `.govuk-width-container` wrapper using Nunjucks
+
+This means you can set specific widths for the:
+
+- [page wrapper](https://design-system.service.gov.uk/styles/layout/#page-wrappers)
+- [header component](https://design-system.service.gov.uk/components/header/)
```

Comment:
It's the containers within the component not the component itself, you can apply this with the Nunjucks option 'containerClasses'

Reply:
I think the overall user task though is that you're setting the width for the wrapper / component? This bit's about what the user's aiming to achieve overall, rather than the details on exactly how to do it (which come further down).

Reply:
There's a difference between a component's width and the container widths which are inside the component, if we don't think that's important that's fine but it may confuse people.